app/views/hyrax/homepage/_featured_fields.html.erb

```diff
@@ -1,16 +1,14 @@
-<h3>
-  <span class="sr-only">Title</span>
-  <%= link_to truncate(featured.title.first, length: 28, separator: ' '), [main_app, featured] %>
-</h3>
 <div>
-  <span class="sr-only">Depositor</span>
-  <%= link_to_profile featured.depositor("no depositor value") %>
-</div>
-<div>
-  <span class="sr-only">File type</span>
-  <%= link_to_facet_list(featured.resource_type, 'resource_type', 'no resource specified') %>
-</div>
-<div>
-  <span class="sr-only">Keywords</span>
-  <%= link_to_facet_list(featured.keyword, 'keyword', 'no keywords specified') %>
+  <div class="featured-item-title">
+    <span class="sr-only">Title</span>
+    <h3><%= link_to featured.title.first, [main_app, featured] %></h3>
+  </div>
+  <div class="featured-item-depositor">
+    <span class="sr-only">Creator</span>
```

Comment:
Shouldn't this say "Depositor"?

Reply:
Ah, yeah, good catch. I changed it elsewhere. Thanks.